recipe_data.csv

```diff
@@ -0,0 +1,17 @@
+id,name,cashCost,carrotsCost,grassCost,fertilizerCost

```

Comment:
We need another column here, "seed_id".  For these, it looks like they'll be the same but that won't always be true.